doc/onboarding/model/form.md

```diff
@@ -0,0 +1,30 @@
+# `form` State
+
+## Definition
+State of a form that includes its values but also :
+ - required field or
+ - ability to submit or not all the form.
+
+```
+{
+    <boolean> disabled: false
```

Comment:
Is there a user case when we need a form that is submittable with errors? I only see the case when a step is not mandatory and the user can skip it. If so, I think it's better to use ```skippable``` or ```mandatory``` as boolean here.